resources/testdriver.js

```diff
@@ -680,6 +680,134 @@
         set_spc_transaction_mode: function(mode, context=null) {
           return window.test_driver_internal.set_spc_transaction_mode(mode, context);
         },
+
+        /**
+         * Cancels the Federated Credential Management dialog
+         *
+         * Matches the `Cancel dialog
+         * <https://fedidcg.github.io/FedCM/#webdriver-canceldialog>`_
+         * WebDriver command.
+         *
+         * @param {WindowProxy} context - Browsing context in which
+         *                                to run the call, or null for the current
+         *                                browsing context.
+         *
+         * @returns {Promise} fulfilled after the cancel request is sent, or rejected
```

Comment:
Does "after the cancel request is sent" mean that there's no guarantee that the request has been processed when the promise resolves? I suspect that actually we resolve the promise when the request has finished, and if so can you update the wording here to give a stronger guarantee?

Reply:
Agreed that the wording here is kind of iffy - rephrased to give a stronger guarantee. Thanks!